packages/manager/modules/iplb/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@ovh-ux/manager-iplb",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud Iplb product",
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "flag-icon-css": "^3.2.0",
+    "lodash": "^4.17.15",
+    "moment": "^2.19"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^7.6.0",
+    "@ovh-ux/ng-at-internet": "^4.0.0",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.4.0",
+    "@ovh-ux/ng-ovh-doc-url": "^1.0.0",
+    "@ovh-ux/ng-ovh-sidebar-menu": "^8.4.4",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-chart.js": "^1.1.1",
+    "angular-translate": "^2.18.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "bootstrap": "~3.3.0",
```

Comment:
You require bootstrap twice. Once as a dependency in ^4.0.0 and here as a peer in `3.3.x`. Are both needed ?